src/transport/GasTransport.cpp

```diff
@@ -275,6 +275,28 @@ void GasTransport::getMixDiffCoeffs(doublereal* const d)
     }
 }
 
+void GasTransport::getUlnDiffCoeffs(doublereal* const d,doublereal lambda)
+{
+    update_T();
+    update_C();
+
+    // update the binary diffusion coefficients if necessary
+    if (!m_bindiff_ok) {
+        updateDiff_T();
+    }
+	 doublereal p      = m_thermo->pressure();
+	 doublereal rho    = m_thermo->density();
+	 doublereal cp     = m_thermo->cp_mass();
+	 doublereal dth    = lambda/(rho*cp);
```

Comment:
In new code, please use `double` instead of the `doublereal` typedef, and do not insert extra space to force alignment of statements.